assets/chat/js/chat.js

```diff
@@ -878,6 +890,9 @@ class Chat {
                         this.regexhighlightcustom.test(
                             message.user.username + " " + message.message
                         )));
+            if (this.settings.get("holidayemotemodifiers")) {
```

Comment:
Go ahead and make this specific to the 14th, the setting (commit) will be reverted on the 15th